// finmath/lib/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "add", {
  enumerable: true,
  get: function get() {
    return _mathArray.add;
  }
});
Object.defineProperty(exports, "sub", {
  enumerable: true,
  get: function get() {
    return _mathArray.sub;
  }
});
Object.defineProperty(exports, "mul", {
  enumerable: true,
  get: function get() {
    return _mathArray.mul;
  }
});
Object.defineProperty(exports, "div", {
  enumerable: true,
  get: function get() {
    return _mathArray.div;
  }
});
Object.defineProperty(exports, "dma", {
  enumerable: true,
  get: function get() {
    return _dma["default"];
  }
});
Object.defineProperty(exports, "sma", {
  enumerable: true,
  get: function get() {
    return _sma["default"];
  }
});
Object.defineProperty(exports, "ema", {
  enumerable: true,
  get: function get() {
    return _ema["default"];
  }
});
Object.defineProperty(exports, "ma", {
  enumerable: true,
  get: function get() {
    return _ma["default"];
  }
});
Object.defineProperty(exports, "wma", {
  enumerable: true,
  get: function get() {
    return _wma["default"];
  }
});
Object.defineProperty(exports, "sd", {
  enumerable: true,
  get: function get() {
    return _sd["default"];
  }
});
Object.defineProperty(exports, "boll", {
  enumerable: true,
  get: function get() {
    return _boll["default"];
  }
});
Object.defineProperty(exports, "macd", {
  enumerable: true,
  get: function get() {
    return _macd["default"];
  }
});
Object.defineProperty(exports, "hhv", {
  enumerable: true,
  get: function get() {
    return _hhvLlv.hhv;
  }
});
Object.defineProperty(exports, "llv", {
  enumerable: true,
  get: function get() {
    return _hhvLlv.llv;
  }
});

var _mathArray = require("math-array");

var _dma = _interopRequireDefault(require("./dma"));

var _sma = _interopRequireDefault(require("./sma"));

var _ema = _interopRequireDefault(require("./ema"));

var _ma = _interopRequireDefault(require("./ma"));

var _wma = _interopRequireDefault(require("./wma"));

var _sd = _interopRequireDefault(require("./sd"));

var _boll = _interopRequireDefault(require("./boll"));

var _macd = _interopRequireDefault(require("./macd"));

var _hhvLlv = require("./hhv-llv");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
